kani-compiler/src/kani_middle/attributes.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//! This module contains code for processing Rust attributes (like `kani::proof`).
+
+use rustc_ast::{AttrKind, Attribute, LitKind};
+
+/// If the attribute is named `kanitool::name`, this extracts `name`
+fn kanitool_attr_name(attr: &Attribute) -> Option<String> {
```

Comment:
nit: can you move the public methods to the top of the file? That's usually the entry point of the logic, so it's easier to read if they are on the top. :)

Reply:
Good idea! Done